Swift Code Challenge/Models/UserModel.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  UserModel.swift
+//  Swift Code Challenge
+//
+//  Created by Luiz Eduardo Mello dos Reis on 29/06/22.
+//
+
+import Foundation
+
+struct ID: Equatable, Hashable, Decodable {
+    let value: String?
+}
+
+struct Name: Equatable, Decodable {
+    let first: String?
+    let last: String?
+}
+
+struct Picture: Equatable, Decodable{
+    let large: String?
+    let medium: String?
+    let thumbnail: String?
+}
+
+struct coordinates: Equatable, Decodable{
```

Comment:
We should always capitalize objects and keep spacing appropriate. 

```suggestion
struct Coordinates: Equatable, Decodable {
```